src/features/GetCommands.ts

```diff
@@ -42,12 +53,14 @@ export class GetCommandsFeature implements IFeature {
 
     public setLanguageClient(languageclient: LanguageClient) {
         this.languageClient = languageclient;
-        vscode.commands.executeCommand("PowerShell.RefreshCommandsExplorer");
+        if (this.commandsExplorerTreeView.visible) {
+            vscode.commands.executeCommand("PowerShell.RefreshCommandsExplorer");
+        }
     }
 
     private CommandExplorerRefresh() {
         if (this.languageClient === undefined) {
-            this.log.writeAndShowError(`<${GetCommandsFeature.name}>: ` +
+            this.log.writeVerbose(`<${GetCommandsFeature.name}>: ` +
```

Comment:
Seems like this log message isn't super accurate `Unable to instantiate, …`.  It probably should be more like `Unable to send getCommand request, …`.